fix: filter column based on cell value

JordanGerada3 opened the pull request:
## Description

- Added value getter to column 

## Why

The filter of the status column('enabled field') was filtering based on the field 'enabled' which was a boolean value. The value we are setting within the cell however, is using a combination or two different fields to set variable values, resulting in the filter function of the component not able to filter based on the correct value. 

## Changelog entry

 - **IdP Management**
   - fixed IdP table status filter to sort by alphabetical order [#1443](https://github.com/eclipse-tractusx/portal-frontend/pull/1443)

## Issue

#1442 

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/admin/Dev%20Process/How%20to%20contribute.md#commits-branches-and-pull-requests-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

Usmanfee commented on src/components/pages/IDPManagement/IDPList.tsx:
```diff
@@ -450,8 +445,17 @@ export const IDPList = ({ isManagementOSP }: { isManagementOSP?: boolean }) => {
           headerName: t('global.field.status'),
           flex: 2,
-          renderCell: ({ row }: { row: IdentityProvider }) =>
+          valueGetter: ({ row }: { row: IdentityProvider }) =>
```
```suggestion
          valueGetter: ({ row }) => getStatus(row.enabled, row.oidc?.clientId),
```

Usmanfee commented:
**Tested Locally**
- ✅ Sorting either Ascending/Descending groups rows based on status.

manojava-gk commented:
kindly add changelog entry to the PR description

JordanGerada3 commented:
> kindly add changelog entry to the PR description

@manojava-gk Changeling added, thank you.

manojava-gk commented:
> > kindly add changelog entry to the PR description
> 
> @manojava-gk Changelog added, thank you.

@JordanGerada3 you miss understood my comment earlier. I meant this -https://github.com/eclipse-tractusx/portal/wiki/Example-of-Changelog-entry-in-PR-description

JordanGerada3 commented:
> > > kindly add changelog entry to the PR description
> > 
> > 
> > @manojava-gk Changelog added, thank you.
> 
> @JordanGerada3 you miss understood my comment earlier. I meant this -https://github.com/eclipse-tractusx/portal/wiki/Example-of-Changelog-entry-in-PR-description

@manojava-gk I understand what you are referring to now. I have added it to the description now and removed it from the log.
